tools/set-version.py

```diff
@@ -0,0 +1,106 @@
+#!/bin/env python3
+#
+# Sets the version number in the UFO metadata and inserts a special version
+# glyph that helps with debugging when getting user reports. Reads VERSION.txt
+# of the format:
+# ```
+# 1.234 Some optional text to go into the name table version field
+# ```
+#
+# See http://silnrsi.github.io/FDBP/en-US/Versioning.html for version semantics.
+#
+# Changing the components or their widths may cause an error during VTT
+# compilation. Build the VTT builds and run this script:
+# ```
+# import fontTools.ttLib
+# import vttLib
+# import glob
+#
+# for ttf in glob.glob("source/*ttf"):
+#   font=fontTools.ttLib.TTFont(ttf)
+#   vttLib.update_composites(font)
+#   font.save(ttf)
+# ```
+# Then redump the VTT source with
+# ```
+# for f in source/*ttf; do python -m vttLib dump $f ${f/ttf/ufo};done
+# ```
```

Comment:
Ideally this code should be moved into a separate script so that its usable.

The comment should say clearly what needs to be done; eg.
```sh
make updateversion
make updatevtt
make updateversion
```

at the moment it's confusing and unclear.